src/ReactionMechanismSimulator.jl

```diff
@@ -1,83 +1,107 @@
-import Logging
-Logging.disable_logging(Logging.Warn)
-
 module ReactionMechanismSimulator
-    using PyCall
-    push!(PyVector(pyimport("sys")["path"]), "")
-    const Chem = PyNULL()
-    const Desc = PyNULL()
-    const molecule = PyNULL()
-    const fragment = PyNULL()
-    const pydot = PyNULL()
-    const chemkin =PyNULL()
-    const species = PyNULL()
-    const reaction = PyNULL()
-    const nasa = PyNULL()
-    const wilhoit = PyNULL()
-    const arrhenius = PyNULL()
-    const falloff = PyNULL()
-    const chebyshev = PyNULL()
-    const solvation = PyNULL()
-    const os = PyNULL()
-    function __init__()
-        copy!(Chem,pyimport_conda("rdkit.Chem","rdkit","rmg"))
-        copy!(Desc,pyimport_conda("rdkit.Chem.Descriptors","rdkit","rmg"))
-        try
-            copy!(molecule,pyimport("rmgpy.molecule"))
-            copy!(chemkin,pyimport("rmgpy.chemkin"))
-            copy!(species,pyimport("rmgpy.species"))
-            copy!(reaction,pyimport("rmgpy.reaction"))
-            copy!(nasa,pyimport("rmgpy.thermo.nasa"))
-            copy!(wilhoit,pyimport("rmgpy.thermo.wilhoit"))
-            copy!(arrhenius,pyimport("rmgpy.kinetics.arrhenius"))
-            copy!(falloff,pyimport("rmgpy.kinetics.falloff"))
-            copy!(chebyshev,pyimport("rmgpy.kinetics.chebyshev"))
-            copy!(solvation,pyimport("rmgpy.data.solvation"))
-            copy!(fragment,pyimport("rmgpy.molecule.fragment"))
-        catch e
-            copy!(molecule,pyimport_conda("molecule.molecule","rmgmolecule","hwpang"))
-            copy!(chemkin,pyimport_conda("molecule.chemkin","rmgmolecule","hwpang"))
-            copy!(species,pyimport_conda("molecule.species","rmgmolecule","hwpang"))
-            copy!(reaction,pyimport_conda("molecule.reaction","rmgmolecule","hwpang"))
-            copy!(nasa,pyimport_conda("molecule.thermo.nasa","rmgmolecule","hwpang"))
-            copy!(wilhoit,pyimport_conda("molecule.thermo.wilhoit","rmgmolecule","hwpang"))
-            copy!(arrhenius,pyimport_conda("molecule.kinetics.arrhenius","rmgmolecule","hwpang"))
-            copy!(falloff,pyimport_conda("molecule.kinetics.falloff","rmgmolecule","hwpang"))
-            copy!(chebyshev,pyimport_conda("molecule.kinetics.chebyshev","rmgmolecule","hwpang"))
-            copy!(solvation,pyimport_conda("molecule.data.solvation","rmgmolecule","hwpang"))
-            copy!(fragment,pyimport_conda("molecule.molecule.fragment","rmgmolecule","hwpang"))
-        end
-        copy!(pydot,pyimport_conda("pydot","pydot"))
-        copy!(os,pyimport_conda("os","os"))
+ENV["JULIA_CONDAPKG_BACKEND"] = "MicroMamba"
+using CondaPkg
+using Logging
+packages = keys(CondaPkg.current_packages())
+
+if !("rmg" in packages) && !("rmgmolecule" in packages)
```

Comment:
I realize that these don't actually work for packages that are installed locally, so we should change the way we test these